.circleci/config.yml

```diff
@@ -586,7 +586,7 @@ jobs:
             ./script/bqetl metadata update \
               sql/
 
-            mkdir -p /tmp/workspace/main-generated-sql/sql
+            mkdir -p /tmp/workspace/main-generated-sql
             cp -r sql/ /tmp/workspace/main-generated-sql/sql
```

Comment:
If the target `sql` directory already exists, the `cp` command ends up creating a new `sql` directory nested inside the existing `sql` directory.